### DISCUSSION TITLE
Does `get_surrounding_cells()` use Moore or von Neumann Neighborhood?

### DISCUSSION THREAD

**Discussion by DreepyYunky** [godotengine/godot-proposals#11207] [2D]
And if one, can you toggle to the other?

**Comment by sockeye-d** [comment]
From the [documentation](https://docs.godotengine.org/en/latest/classes/class_tilemaplayer.html#class-tilemaplayer-method-get-surrounding-cells):
> Any neighboring cell is one that is touching edges, so for a square cell 4 cells would be returned, for a hexagon 6 cells are returned.

**Comment by tetrapod00** [comment]
There is a proposal to support both: https://github.com/godotengine/godot-proposals/issues/3973

**Reply by DreepyYunky** [reply]
That's cool.
